_data/speakers.yml

```diff
@@ -658,17 +658,20 @@ In my spare time I like to go mountain biking, bouldering, do some woodworking o
   name: ali-akbar-septiandri
   subtitle: Speaker
   twitter: aliakbars
-  github:
-  web:
+  github: aliakbars
+  web: https://aliakbars.id
   image: ali-akbar-septiandri.jpg
   thumbnail: ali-akbar-septiandri.jpg
   alt: Ali
   topic: My Journey in Learning and Relearning Bayesian Statistics
-  proposal_summary: My journey in learning (and relearning) Bayesian methods as a computer scientist
-  discourse_link:
-  youtube_link:
-  description: A data scientist and a lecturer.
-    Learning/teaching data science, machine learning, and artificial intelligence.
+  proposal_summary: In this talk, I will share my journey in learning (and relearning) Bayesian methods as a computer scientist
+    and the hurdles I've found along the way. I will also share some of the resources that I've found useful in my journey.
+  discourse_link:
+  youtube_link: https://www.youtube.com/channel/UCJvg7v0hwrXwaZP17Gh_O5w/
```

Comment:
This field is not used yet, but unlike github/twitter/web links, it should be a link to the posted talk, similarly to how `discourse_link` should not be a link to your profile but to the post where the discussion about your talk is happening.